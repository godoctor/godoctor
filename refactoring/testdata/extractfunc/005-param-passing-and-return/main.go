//<<<<<extract,10,2,14,15,Foo,pass
package main

import "fmt"

func main() {
	a := 3
	b := a
	c := 1
	for a <= b {
		a += b
	}
	x := a + b
	fmt.Println(x)
	z := a + x + c
	fmt.Println(z)
}
