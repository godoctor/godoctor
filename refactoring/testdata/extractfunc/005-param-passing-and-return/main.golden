//<<<<<extract,10,2,14,15,Foo,pass
package main

import "fmt"

func main() {
	a := 3
	b := a
	c := 1
	a, x := Foo(a, b)
	z := a + x + c
	fmt.Println(z)
}
func Foo(a int, b int) (int, int) {
	for a <= b {
		a += b
	}
	x := a + b
	fmt.Println(x)
	return a, x
}
